### ISSUE TITLE
0006906:
[softaculous] My Hosting Provider upgraded the server to PHP 5.3.15 and Tine 2.0 stopped working

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#3501]
**Reported by joejac on 9 Aug 2012 03:16**

Hello,
Yesterday my Hosting Provider upgraded the server to PHP 5.3 and Tine stopped working, with an error &quot;Service Unavailable&quot; I appreciate a lot your help because I can not use Tine, this is their message:

&quot;Hello Jose,

Thank you for your patience. The error is due to the cms &#039;Tine&#039; utilizing an outdated version of Zend Optimizer. We have recently updated to the latest version of PHP 5.3 which no longer provides Zend Optimizer support, but now instead utilizes Zend Guard Loader.

Unfortunately, there is nothing we can do to assist with this issue, as it is not possible to run the Zend Optimizer loader on PHP 5.3 and later. You will need to contact the developer of the &#039;Tine&#039; cms and ask for a Zend Guard encoded version that will work on PHP 5.3 or later or get a version of the script encoded for IonCube.

Please let us know if you have any further questions.

Regards,
-----
Server Analyst II
SurpassHosting.com, LLC&quot;

**Steps to reproduce:** Upgrade the Server to the latest version of PHP 5.3.

Please note that everybody that makes the upgrade will experience same problem with Tine 2.0: &quot;Service Unavailable&quot;

**Additional information:** Apache version 	2.2.22
PHP version 	5.3.15
MySQL version 	5.0.95-community
Architecture 	x86_64
Operating system linux
Kernel version 	2.6.18-308.11.1.el5
cPanel Version 	11.32.3 (build 21)

http://www.softaculous.com/apps/erp/Tine

**Comment by Gloirin** [comment]
**Comment posted by lkneschke on 9 Aug 2012 04:22**

Tine 2.0 is NOT using Zend Optimizer.  We don&#039;t provide any downloads which require Zend Optimizer or any similar product.

Maybe someone else installed you an encoded version of TIne 2.0. If that&#039;s the case you should be able to solve your problem by downloading the latest version of Tine 2.0 from tine20.org again.

**Comment by Gloirin** [comment]
**Comment posted by joejac on 9 Aug 2012 15:30**

Thanks a lot lkneschke for your fast response.
It is strange because what I did 3 weeks ago was:
1. I installed Tine 2.0 using Softaculous, provided by my Web Hosting Company, but it did not worked, so I uninstalled using same Softaculous.
2. I downloaded Tine 2.0 from your website, installed manually in the same server account and it worked.
Maybe Sotfaculous installed some extra scripts and they were not successfully uninstalled.
I will tell my web hosting company, hope they can find the issue, if not I will delete the account and install Tine 2.0 as you say.
Best regards and sorry to post this, but I did not know Tine 2.0 do not use Zend Optimizer.
Jose
